.gitignore

```diff
@@ -24,4 +24,5 @@ Desktop.ini
 $RECYCLE.BIN/
 
 # Mac desktop service store files
-.DS_Store
\ No newline at end of file
+.DS_Store
+/exercises/02-publish-subscribe/after/Divergent.Customers/Divergent.Customers.csproj.user
```

Comment:
This change should be reverted, or a line should be added to ignore _all_ `*.csproj.user` files (should be in a separate commit/PR).